source/SIL.AppBuilder.Portal/src/routes/(authenticated)/tasks/[product_id]/components/SortTable.svelte

```diff
@@ -0,0 +1,62 @@
+<script lang="ts">
+  import SortDirectionPicker from "./SortDirectionPicker.svelte";
+  export let items: {[key: string]: any}[];
+
+  let current = "";
+  let desc: {[key: string]: boolean} = {};
+  $: console.log(desc);
```

Comment:
remove console.log?

Reply:
Removed. Since I know it works correctly, I shouldn't need that there anymore.